### ISSUE TITLE
[Bug] i.sentinel.coverage: cannot download Sentinel-1 SLC data 

### ISSUE THREAD

**Issue by neteler** [OSGeo/grass-addons#447]
**Name of the addon**
i.sentinel.coverage

**Describe the bug**

I do not manage to download Sentinel-1 SLC data:

```
GRASS nc_spm_08_grass7/user1 >
i.sentinel.coverage settings=~/.sentinel_esa output=s2names.txt producttype=SLC minpercent=95 area=county_durham_dp1000 start=2020-10-01 end=2021-01-31
Retrieving Sentinel footprints from ESA hub ...
ERROR: The name of the scene must have a format of e.g.
       S2A_MSIL1C_YYYYMMDDT155901_N0206_R097_T17SPV_20180822T212023
```

Maybe I am missing something (wrong parameter)?

**Expected behavior**
`i.sentinel.coverage` should not attempt to download S2 when SLC is requested by user.

**System description (please complete the following information):**
 - Fedora 33
 - GRASS GIS version 7.8.git

**Comment by griembauer** [comment]
This bug is due to the fact that SLC scenenames have one more `_` (underscore), which confuses the identification of scene specifications by `' '.split('_')`.
This is fixed in #453

**Comment by neteler** [comment]
Thanks for the fix (hint: using stop-words in PRs closes issues automatically like `fixes #1234`).
